std/parallelism.d

```diff
@@ -3232,7 +3286,8 @@ public:
     */
     int priority() @property @trusted
     {
-        return (size == 0) ? core.thread.Thread.PRIORITY_MIN :
+        //import core.thread : Thread;
```

Comment:
Is this here to make clear where `Thread` is coming from or just something you forgot to remove?  If the former, I don't think it's necessary.